lib/ybe.gd

```diff
@@ -99,8 +99,7 @@ DeclareAttribute("DerivedRightRack", IsYB);
 #! @ExampleSession
 #! gap> cs := SmallCycleSet(4,13);;
 #! gap> yb := CycleSet2YB(cs);;
-#| gap> DehornoyClass(yb);
-#! 2
+#| gap> DehornoyClass(yb);;
```

Comment:
Alternatively, you could keep the first line as-is, and just change the second line to `#| 2`

Reply:
That's the least problem. Some examples do not work and trigger a recursion depth trap. @vendramin we need to investigate.